src/legacy/core_plugins/kibana/public/home/components/welcome.tsx

```diff
@@ -43,6 +43,7 @@ import { SampleDataCard } from './sample_data';
 interface Props {
   urlBasePath: string;
   onSkip: () => void;
+  optInSeen: () => any;
```

Comment:
NIT: naming. This is an event handler. `optInSeen` -> `onOptInSeen` (like `onSkip` here). Same goes for propagaging from `home.js`